core/db/curibio/alembic/versions/39405902fa31_add_running_to_status_enum_in_jobs_.py

```diff
@@ -0,0 +1,32 @@
+"""add running to status enum in jobs_result
+
+Revision ID: 39405902fa31
+Revises: 827f22860c04
+Create Date: 2023-04-01 11:10:12.975445
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "39405902fa31"
+down_revision = "827f22860c04"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute("""ALTER TYPE "JobStatus" ADD VALUE 'running'""")
+
+
+def downgrade():
```

Comment:
I think you'll also want to change any jobs with the running status to pending

Reply:
Yeah better to ensure you're right